CONTRIBUTING.md

```diff
@@ -21,16 +21,20 @@ This project adheres to the llm-d [Code of Conduct](https://github.com/llm-d/llm
 
 We welcome contributions to the llm-d-modelservice chart! If you have a bug fix, feature request, or improvement, please submit a pull request (PR) to the repository.
 
-Before submitting a pull request, please ensure that you have the following dependencies installed and set up:
+Before submitting a pull request, please ensure that you have the following tools installed. You can install them via `make tools`.
 
 - [Helm](https://helm.sh/)
+- [Chart Testing](https://github.com/helm/chart-testing)
 <!-- TODO after the pre-commit supported 
 - [Helm docs](https://github.com/norwoodj/helm-docs) 
 - [pre-commit](https://pre-commit.com/) -->
 
+To avoid system-wide impact, `make tools` installs these tools in the `PROJECT_DIR/bin` directory. You may need to copy them to your `/usr/local/bin` directory to run them directly.
```

Comment:
I almost feel like it's the developer's responsibility to resolve versioning conflicts... did this happen to you when you first started contributing? Just want to understand the motivation behind this a little better. 

Also this seems to download regardless if the tool is found locally. Is this behavior intended?